python/tskit/tables.py

```diff
@@ -32,15 +32,14 @@
 import numpy as np
 
 import _tskit
+import tskit
+import tskit.provenance as provenance
+import tskit.util as util
 
 # This circular import is ugly but it seems hard to avoid it since table collection
```

Comment:
Is this comment still relevant? I'm not seeing the issue with the import.

Reply:
No, I think it's fine. I used to work with someone who **really** hated circular imports so I've been apologetic about it since. It's nothing to worry about though, so yeah, let's dump the comment.

Reply:
Fixed